### ISSUE TITLE
Extend  README.md

### ISSUE THREAD

**Issue by samtuke** [templateria/php-spamassassin#5]
The classes look great, but the README file doesn't give any clues as to the purpose of this library, or how to use it.

Please could you add a few lines explaining the intention behind his library, and some hints as to how to get started with it (e.g. should client be used as a public interface to the lib internals?). Thanks!

**Comment by jstrouse** [comment]
Here's a small sample starter program.....

<?php
require_once 'vendor/autoload.php';

$params = array(
    "hostname" => "localhost",
    "port" => "783",
    "user" => "ubuntu",
);

$sa = new Spamassassin\Client($params);

$message = file_get_contents('testemail.txt');

$result = $sa->getSpamReport($message);
echo $result->message . "\n";
echo $result->score . "\n";
?>

**Comment by samtuke** [comment]
Thanks! So this class interacts with spam assassin and provides a PHP interface to it, if you will?

**Comment by jstrouse** [comment]
Correct.  I'm using PHPMailer and I generated a sample email text and ran it through SpamAssassin to provide feedback to users if the email their about to send looks like spam.

**Comment by yellow1912** [comment]
This project is very interesting, it can give way to setup your own spam detection server, not just for email but for all types of messages. I have yet to find a reliable spam detection on the market and this can be a good alternative.
